packages/hadron-build/lib/target.js

```diff
@@ -715,10 +715,8 @@ class Target {
         },
         bin: this.productName,
         requires: [
-          'libXScrnSaver',
           'gnome-keyring',
           'libsecret',
-          'GConf2'
```

Comment:
Do you know what removed the dependency on libgconf?

Reply:
We think electron dropped dependency on it some time ago (because chromium also removed this dependency)

https://github.com/electron/electron/pull/19498